### PR TITLE
Tmforum

### PR DESCRIPTION
**Pull request by wistefan**

### CODE REVIEW

File: charts/tm-forum-api/values.yaml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+   
+  # -- a list of additional env vars to be set, check the tm-forum api docu for all available options
+  additonalEnvVars: []
+
+  # -- configuration to be used for the image of the container
+  image:
+    # -- repository to get the container from
+    repository: quay.io/wi_stefan
+    # -- tag to be used, most of the time the apis will use the same version
+    tag: 0.4.1
```
Comment (dwendland) [inline charts/tm-forum-api/values.yaml:144]: Do all images always have the same version tag?

File: charts/business-api-ecosystem/README.md [hunk 2]
```diff
@@ -273,8 +273,6 @@ A Helm chart for running the FIWARE business API ecosystem (FIWARE Marketplace)
 | siop.ccs.credentials | list | `[{"trustedIssuersLists":["https://til.fiware.dev"],"trustedParticipantsLists":["https://tir.fiware.dev"],"type":"VerifiableCredential"}]` | Credential configuration to be registered |
 | siop.ccs.endpoint | string | `"http://credentials-config-service:8080"` | Endpoint of the CCS |
 | siop.clientId | string | `"marketplace-client"` |  |
-| siop.credentialTypes[0] | string | `"VerifiableCredential"` |  |
```
Comment (dwendland) [inline charts/business-api-ecosystem/README.md:276]: Aren't there also changes to the values.yaml?

File: charts/tm-forum-api/values.yaml (hunk 1 above)
Comment (wistefan) [reply charts/tm-forum-api/values.yaml:144]: Not neccessarily, thats why you can overwrite them individually

Comment (dwendland) [reply charts/tm-forum-api/values.yaml:144]: I did overlook that. The added description in the Readme is good.

**Review by dwendland — APPROVED** [review]

**Review by beknazaresenbek — APPROVED** [review]

**Pull request merged**